runtime/shared_common/j9shr.tdf

```diff
@@ -2935,3 +2935,10 @@ TraceEvent=Trc_SHR_INIT_j9shr_findGCHints_Event_Found Overhead=1 Level=4 Templat
 TraceEvent=Trc_SHR_CM_storeSharedData_OverwriteExisting Overhead=1 Level=4 Template="CM storeSharedData: Existing data in the shared cache has been overwritten (result %p, data->address %p, foundDatalen %zu)."
 TraceEvent=Trc_SHR_CM_updateLocalHintsData_OverwriteHeapSizes Overhead=1 Level=4 Template="CM updateLocalHintsData: Existing hints (heapSize1=%zu, heapSize2=%zu) in the shared cache will be overwritten to (heapSize1=%zu, heapSize2=%zu)."
 TraceEvent=Trc_SHR_CM_updateLocalHintsData_WriteHeapSizes Overhead=1 Level=4 Template="CM updateLocalHintsData: Will write hints (heapSize1=%zu, heapSize2=%zu) to shared cache."
+TraceExit=Trc_SHR_OSC_getCacheStatistics_LayerNumberTooBig NoEnv Overhead=1 Level=1 Template="OSCache::getCacheStatistics: Exit - layer number is greater than the maxium value"
```

Comment:
This should be TraceException

Reply:
Removed this trace point. It is replaced with an assertion.